flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -231,6 +234,23 @@ public void setForceCheckpointing(boolean forceCheckpointing) {
 	}
 
 	/**
+	 * This determines the behaviour of tasks if there is an error in their checkpointing. If this returns true,
+	 * tasks will fail as a reaction. If this returns false, task will only decline the failed checkpoint.
+	 */
+	public boolean isFailTasksOnCheckpointingErrors() {
```

Comment:
I think we typically don't talk about tasks in the user facing APIs. This could be `isFailOnCheckpointingErrors()`, for example.